frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -274,26 +284,24 @@ export const PersonalDashboard = () => {
                 </SpacedGridItem>
                 <SpacedGridItem item lg={4} md={1}>
                     {onboardingCompleted ? (
-                        <ProjectSetupComplete project={activeProject} />
+                        <ProjectSetupComplete project={activeProject!.id} />
```

Comment:
Is there a plan to remove "!" ?

Reply:
Yes, that's also one! There's some TS inference stuff going on here, I think